testing/resources/renv_hooks_repo/DESCRIPTION

```diff
@@ -0,0 +1,18 @@
+Package: gli.clu
```

Comment:
The `DESCRIPTION` file is basically the equivalent of `setup.py`, `renv.lock` is the equivalent of the conda `environment.yml`. Not sure if one or both should be listed in `LOCAL_RESOURCES` as this was introduced here:
https://github.com/pre-commit/pre-commit/pull/1232/files#diff-a52ef0cf627ce783529c259e63d4ea82b77520936a2455cbf4f90ad2fb36d27dR176

Reply:
ah `LOCAL_RESOURCES` is to allow an `additional_dependencies`-only `repo: local` hook -- it seems like additional_dependencies is supported for R so we should probably make an empty one?

Reply:
Ok, thanks for the explanation. I added a template for `renv.lock` in cf88089e717c766afb456b5f4737bd73dbb90c24. However, there is an R version hard-coded in it which will get outdated. As far as I can tell from a quick experiment, the docs and issues in https://github.com/rstudio/renv, there is no indication that the R version recorded in the lock file will be used at `renv::restore()`, which we use in `renv.install_environment()`. Even removing seems fine (I left it there anyways). But that might still be a bit brittle.

Reply:
Also, should we add a unit test for a local hook with additional dependencies to check if things work as expected? I don't think that's currently covered.

Reply:
yeah it would probably be good to have one unit test demoing a `local` hook via `additional_dependencies`